open_city_profile/settings.py

```diff
@@ -214,3 +215,7 @@
                 "Please create a %s file with random characters to generate your secret key!"
                 % secret_file
             )
+
+# Graphene
+
+GRAPHENE = {"SCHEMA": "profiles.schema.schema"}
```

Comment:
well, we should have one project-wide schema in `open_city_profile/schema.py`, which would gather up all the schema from different apps, like [we have in venepaikka](https://github.com/City-of-Helsinki/berth-reservations/blob/develop/berth_reservations/schema.py#L15)